models/model.js

```diff
@@ -101,4 +101,17 @@ exports.selectUsers = (req, res) => {
     .then((results) => {
         return results.rows;
     })
-}
\ No newline at end of file
+}
+
+exports.removeComments  = (ID) => {
```

Comment:
When we delete a comment we shouldn't be sending a response body. Perfectly fine to check the rowCount for error handling though.